chart/templates/triggerer/triggerer-deployment.yaml

```diff
@@ -0,0 +1,212 @@
+      {{- end }}
+      initContainers:
+      {{- if and $persistence .Values.triggerers.persistence.fixPermissions }}
+        - name: volume-permissions
+          resources:
+{{ toYaml .Values.triggerers.resources | indent 12 }}
+          image: {{ template "airflow_image" . }}
+          imagePullPolicy: {{ .Values.images.airflow.pullPolicy }}
+          command:
+            - chown
+            - -R
+            - "{{ .Values.uid }}:{{ .Values.gid }}"
+            - {{ template "airflow_logs" . }}
+          securityContext:
+            runAsUser: 0
+          volumeMounts:
+            - name: logs
+              mountPath: {{ template "airflow_logs" . }}
+      {{- end }}
+        - name: wait-for-airflow-migrations
+          resources:
+{{ toYaml .Values.triggerers.resources | indent 12 }}
+          image: {{ template "airflow_image" . }}
+          imagePullPolicy: {{ .Values.images.airflow.pullPolicy }}
+          args:
+          {{- include "wait-for-migrations-command" . | indent 10 }}
+          envFrom:
+          {{- include "custom_airflow_environment_from" . | default "\n  []" | indent 10 }}
+          env:
+          {{- include "custom_airflow_environment" . | indent 10 }}
+          {{- include "standard_airflow_environment" . | indent 10 }}
+        {{- if .Values.triggerers.extraInitContainers }}
+        {{- toYaml .Values.triggerers.extraInitContainers | nindent 8 }}
+        {{- end }}
+      containers:
+        - name: triggerer
+          image: {{ template "airflow_image" . }}
+          imagePullPolicy: {{ .Values.images.airflow.pullPolicy }}
+          {{- if .Values.triggerers.command }}
+          command: {{ tpl (toYaml .Values.triggerers.command) . | nindent 12 }}
+          {{- end }}
+          {{- if .Values.triggerers.args }}
+          args: {{ tpl (toYaml .Values.triggerers.args) . | nindent 12 }}
+          {{- end }}
+          resources:
+{{ toYaml .Values.triggerers.resources | indent 12 }}
+          volumeMounts:
+{{- if .Values.triggerers.extraVolumeMounts }}
+{{ toYaml .Values.triggerers.extraVolumeMounts | indent 12 }}
+{{- end }}
+            - name: logs
+              mountPath: {{ template "airflow_logs" . }}
+            - name: config
+              mountPath: {{ template "airflow_config_path" . }}
+              subPath: airflow.cfg
+              readOnly: true
+            {{- if .Values.airflowLocalSettings }}
+            - name: config
+              mountPath: {{ template "airflow_local_setting_path" . }}
+              subPath: airflow_local_settings.py
+              readOnly: true
+            {{- end }}
+          envFrom:
+          {{- include "custom_airflow_environment_from" . | default "\n  []" | indent 10 }}
+          env:
+          {{- include "custom_airflow_environment" . | indent 10 }}
+          {{- include "standard_airflow_environment" . | indent 10 }}
+{{- if $persistence }}
+        - name: triggerer-log-groomer
```

Comment:
Right now the triggerer doesn't start the log server, should it?

I haven't really dug in here, but if we are storing logs locally, thus needing a groomer, don't we need to be able to get them?

Reply:
Good point - should we surface the logs of triggerer in the UI somehow? are there any useful logs for DAG writers and ones that can be connected to particular task runs?

Reply:
Me and Andrew G had a talk this morning about it -- currently we don't store logs and there isn't much useful data in Triggerer logs but we will add more observability aroud Triggerer -- most likely somehow show it with Task Logs or a something similar, we haven't yet finalized.